### ISSUE TITLE
CycleHunter automation: periodical runs and notification via e-mail

### ISSUE THREAD

**Issue by EduardoATpt** [SIDN/CycleHunter#19]
Hello,

It would be good to add a script that runs periodically on the crontab and if it finds a Cycle that sends an email.

Thank you.

**Comment by gmmoura** [comment]
I agree with @EduardoATpt : many TLD ops would like to automate this, and have an e-mail based notification system.
@subbink : do you have some code for this?

btw, @EduardoATpt , I renamed it to to expand a bit the scope of the issue

**Comment by subbink** [comment]
@gmmoura : I do not have any code which can be shared at the moment.

**Comment by christian-2** [comment]
FYI we do this with `cronjob/cyclehunter` in a Kubernetes cluster. The job delivers notifications via Kafka, but it could be email as well. (BTW running the container as root is unnecessary, we are therefore also patching `USER nobody` into `Dockerfile`.)

**Comment by seb-at-nzrs** [comment]
Thanks Christian. Would you be willing to contribute the changes or configuration to run in Kubernetes?

**Comment by christian-2** [comment]
@seb-at-nzrs Yes, if you can provide a feature branch (e.g. `feat/k8s-kafka`) upstream I can prepare a PR with the following files. That PR perhaps should not go directly into `master`, because maintainers may want to take a closer look regarding how to specifically support both scenarios (without and with automation) in a single code base, i.e. branch.

* `CycleHunter.py` (patched)
* `Dockerfile` (patched)
* `k8s/cyclehunter.yaml.template`
* `k8s/entrypoint.sh`
